docs/topics/db/transactions.txt

```diff
@@ -176,6 +176,26 @@ Django provides a single API to control database transactions.
         If you catch exceptions raised by raw SQL queries, Django's behavior
         is unspecified and database-dependent.
 
+    .. note::
```

Comment:
Generally we prefer `.. admonition:: descriptive title goes here` rather than `.. note::`. It makes things more skimmable.